cupy/core/raw.pyx

```diff
@@ -24,9 +24,12 @@ cdef class RawKernel:
         options (str): Compiler options passed to NVRTC. For details, see
             https://docs.nvidia.com/cuda/nvrtc/index.html#group__options.
         backend (str): Either `nvrtc` or `nvcc`. Defaults to `nvrtc`
+        translate_cucomplex (bool): Whether the CUDA source includes the
+            "cuComplex.h" header or not. Defaults to False.
```

Comment:
Please describe a behavior a bit more so that users can understand that the code is modified by CuPy. I want avoid surprising users 😉

Reply:
Haha OK I agree. This is done.